File: problems/prob04/prob4.rb
def rand_print(num)
	array = []
	(0..num).each do |x|
		if x % 2 == 0
			array << x
		end
	end
	return array
end

rand_num = rand(1..100)
puts "The number is " + rand_num.to_s + "\n\n"
print rand_print(rand_num)

File: problems/prob04/prob4.py
from random import randrange


def rand_print(num):
    array = []
    for x in range(0, num + 1):
        if x % 2 == 0:
            array.append(x)
    return array


rand_num = randrange(100)
print("The number is " + str(rand_num))
print("")
print(rand_print(rand_num))
